stylesheet/components/home.css

```diff
@@ -0,0 +1,216 @@
+
+body{
+     font-family: 'Poppins', sans-serif;
+}
+.home-name{
+     width: 125px;
+}
+.navbar{
+     display: flex;
+     align-items: center;
+     padding: 20px;
+}
+nav{
+     flex: 1;
+     text-align: right;
+}
+nav ul{
+     display: inline-block;
+     list-style-type: none;
+}
+
+nav ul li{
```

Comment:
just a small suggestion to declare a separate class and use it in the HTML instead of styling directly the HTML default tag